src/test/java/com/wikia/webdriver/testcases/discussions/CreatingReplyTests.java

```diff
@@ -94,9 +94,8 @@ private void anonymousUserOnReplyEditorClickIsRedirectedTo(final ReplyCreator re
   }
 
   private void assertThatUserCanCreateReply(PostDetailsPage page, ReplyCreator replyCreator) {
-    replyCreator.click().clickGuidelinesReadButton();
-    final String text = TextGenerator.createUniqueText();
-    replyCreator.add(text).clickSubmitButton();
+    String text = TextGenerator.createUniqueText();
+    replyCreator.startReplyCreationWith(text).clickSubmitButton();
```

Comment:
what about one-liner

Reply:
I need that value in the line below for checking whether the post with this text was added